config/rbac/service_account.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: controller-manager
```

Comment:
why not controller manager? I believe other CAPI providers have separate docker images for a controller vs manager. In our case, they're combined. Shouldn't the name reflect that?

Reply:
I was following along with the Cluster Api book's upgrade destructions. They used the name manager, and I wasn't sure whether it was relevant or not.

This change wasn't intended for the final PR unless it was necessary.

I just swapped it back as it wasn't.